### ISSUE TITLE
Please mention use of tun2socks as required by license

### ISSUE THREAD

**Issue by ambrop72** [shadowsocks/shadowsocks-android#317]
Hello,

I appreciate that you use my software in this product. But, please, respect the license, which includes the following point.

```
    * Redistributions in binary form must reproduce the above copyright
      notice, this list of conditions and the following disclaimer in the
      documentation and/or other materials provided with the distribution.
```

I could not find any mention of use of tun2socks in the readme, the Google Play page, or the application itself. I think it would be enough it you put together a list of used software and associated licences or links, and link to that list from all the three mentioned places.

Note that this general issue also applies to other software bundled with shadowsocks-android but it's not my place to be bothered by that ;) Though, remember that tun2socks itself comes with a copy of lwIP.

Thank you.

**Comment by madeye** [comment]
Thanks, fixed via 44e1b9d.

**Comment by ambrop72** [comment]
This is good, thank you!
